src/main.ts

```diff
@@ -1,10 +1,18 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from '@main/app.module';
 import { ValidationPipe } from '@nestjs/common';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   app.useGlobalPipes(new ValidationPipe({ transform: true }));
+  const config = new DocumentBuilder()
```

Comment:
Hello, even if all these fields are going to be used on this particular file, we can always extract them as constants.

Reply:
what do you mean? export the config variable?